crates/sui-json-rpc/src/governance_api.rs

```diff
@@ -431,7 +432,8 @@ async fn exchange_rates(
             })
             .collect::<Result<Vec<_>, _>>()?;
 
-        rates.sort_by(|(a, _), (b, _)| a.cmp(b).reverse());
```

Comment:
Is line 466/467 attempting to do the equivalent of this?

Reply:
The loop in `backfill_rates` does backfilling + sorting in ascending order by epoch number. And line 484 reverses the vector so it's descending order.